config/locales/multi_factor_authentication/en.yml

```diff
@@ -8,4 +8,9 @@ en:
       backup if you lose one of your chosen authentication devices.
     method_confirmation:
       face_id: Face ID has been added to your account
+    second_method_warning: 
+      link: Add a second authentication method
+      text_html: "%{link}. You will have to
+      delete your account and start over if you lose your only authentication
+      method."
```

Comment:
I would consider moving the HTML outside of this translation, maybe even putting the period inside the link copy?